ext/stringio/stringio.c

```diff
@@ -494,9 +530,23 @@ strio_get_string(VALUE self)
 
 /*
  * call-seq:
- *   strio.string = string  -> string
+ *   string = string -> string
+ *
+ * Assigns the underlying string as +string+;
```

Comment:
Might be interesting to mention that the cursor is set to 0.

Reading:

```ruby
irb(main):013:0> io = StringIO.new("hello")
=> #<StringIO:0x0000000108cbd208>
irb(main):014:0> io.read
=> "hello"
irb(main):015:0> io.pos
=> 5
irb(main):016:0> io.string = "foo"
=> "foo"
irb(main):017:0> io.pos
=> 0
irb(main):018:0> io.read
=> "foo"
```

Writing:
```ruby
irb(main):034:0> io = StringIO.new("hello")
=> #<StringIO:0x0000000108d4e280>
irb(main):035:0> io.string = "hello world 123"
=> "hello world 123"
irb(main):036:0> io.write("foo")
=> 3
irb(main):037:0> io.string
=> "foolo world 123"
```

Reply:
Added.